lib/rubocop/cop/style/script_permission.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks if a file which has a shebang line as
+      # its first line is granted execute permission.
+      class ScriptPermission < Cop
+        MSG = 'Grant script file execute permission.'.freeze
```

Comment:
Maybe we should mention the file name here, as this wording might convince some people. I'd also use working like `Script file X doesn't have execute permission.` (or something similar).